screens/Home/index.tsx

```diff
@@ -46,18 +51,25 @@ export default function HomeScreen() {
   return (
     <View style={styles.container}>
       <PerlinNoise isOn={isOn} color1="#281713" color2="blue" />
-      <View style={{ width: '100%' }}>
+      <View style={{ width: "100%" }}>
         <TouchableOpacity style={styles.iconContainer}>
           <Settings />
         </TouchableOpacity>
       </View>
+      <TouchableOpacity>
+        <CustomButton
+          title="Testing"
+          variant="primary"
+          onPress={() => navigation.push("Testing")}
+        />
+      </TouchableOpacity>
```

Comment:
why not use just the customButton only since it's already a TouchableOpacity?
```suggestion
        <CustomButton
          title="Testing"
          variant="primary"
          onPress={() => navigation.push("Testing")}
        />
  
```